tests/test_json.py

```diff
@@ -63,12 +69,14 @@ async def test_crud(bind):
     # Reload and test updating both JSON and regular property
     u = await User.get(u.id)
     await u.update(
-        age=User.age - 2, balance=200.15, realname="daisy", nickname="daisy.nick"
+        age=User.age - 2, balance=200.15, realname="daisy", nickname="daisy.nick", height=185, weight=75
```

Comment:
@daichen-daisy you're now 75kg and 185cm 🤣 just kidding, this is cool.

(trivial) Could you please also `black` this file? This line seems to be too long.